frontend/src/components/navigation/navbar.tsx

```diff
@@ -7,39 +7,8 @@ import './navbar.css';
 
 export default function ButtonAppBar() {
     return (
-        <header className="navbar">
-            <div className="title">You’re not alone on your  journey, Haato</div>
-            <div className="icons">
-                {[
-                    {
-                        externalLink: true,
-                        link: 'https://github.com/Manotomo-Alliance-Support-Squad/WWS',
-                        altText: "github",
-                        iconFunc: () => <InfoIcon/>
-                    },
-                ].map((obj, idx) => {
-                    // For accessibility purposes
-                    let buttonAltText = (obj.altText ?? "");
-                    if (obj.externalLink) {
-                        return (
-                            <IconButton target="_blank" rel="noopener noreferrer" href={obj.link}
-                                key={idx} className="button inactive-page-icon" aria-label={buttonAltText}
-                            >
-                                {obj.iconFunc()}
-                            </IconButton>
-                        );
-                    } else {
-                        return (
-                            <NavLink key={idx} to={obj.link} className='inactive-page-icon' activeClassName='active-page-icon'>
-                                <IconButton key={idx} className="button" aria-label={buttonAltText}>
-                                    {obj.iconFunc()}
-                                </IconButton>
-                            </NavLink>
-                        );
-                    }
-                })}
-                <LanguageSwitchButton/>
-            </div>
+        <header className="navbar text-center">
```

Comment:
Did we ant to use altnav buttons for games and credits up near the top?

Reply:
well, I was thinking that we could have two altnav buttons. Since we didn't have the game page yet, I thought it would probably be fine to just leave altnav near the bottom since I thought that's where we probably would be leaving the credits button anyways. We can change it again once we get the games and add the games page